doc/en/components/notifications/banner.md

```diff
@@ -270,7 +268,7 @@ banner.addEventListener('igcClosing', (event) => {
 const bannerRef = useRef<IgrBanner>(null);
 
 useEffect(() => {
-    bannerRef.current.nativeElement.addEventListener('igcClosing', (event) => {
+    bannerRef.current.addEventListener('igcClosing', (event) => {
```

Comment:
This is not needed. You can bind to the `onClosing` and `onClosed` events to react banner component itself. 
![Screenshot 2025-06-10 at 16 43 36](https://github.com/user-attachments/assets/f7f9ebca-8026-4def-891b-f80f8e1b28fc)
As you can see here they are overriden. Please test this as well.

Reply:
PS. probably the text should be updated as well because it is mentioning `igcClosing` and `igcClosed`